src/Drawer/Drawer.spec.js

```diff
@@ -0,0 +1,37 @@
+/* eslint-env mocha */
+import React from 'react';
+import {shallow} from 'enzyme';
+import sinon from 'sinon';
+import Drawer from './Drawer';
+import getMuiTheme from '../styles/getMuiTheme';
+
+describe('<Drawer />', () => {
+  const muiTheme = getMuiTheme();
+  const shallowWithContext = (node) => shallow(node, {context: {muiTheme}});
+
+  describe('propTypes', () => {
+    // Since react will console.error propType warnings, that which we'd rather have
+    // as errors, we use sinon.js to stub it into throwing these warning as errors
+    // instead.
+    before(() => {
```

Comment:
What about using this [partern](https://github.com/callemall/material-ui/blob/next/test/utils/setup.js#L4) instead? That's what we are doing in the `next` branch.